exercises/concept/vehicle-purchase/.docs/introduction.md

```diff
@@ -0,0 +1,104 @@
+# Introduction
```

Comment:
Is this file auto-generated from the `tpl` file? By what tool? Assuming it's auto generated, I'll skip reviewing it.